core/proto.go

```diff
@@ -144,10 +144,15 @@ func ParSignedDataToProto(data ParSignedData) (*pbv1.ParSignedData, error) {
 		return nil, errors.Wrap(err, "marshal share signed data")
 	}
 
+	var sigs [][]byte
+	for _, sig := range data.Signatures() {
+		sigs = append(sigs, sig)
+	}
+
 	return &pbv1.ParSignedData{
-		Data:      d,
-		Signature: data.Signature(),
-		ShareIdx:  int32(data.ShareIdx),
+		Data:       d,
+		Signatures: sigs,
```

Comment:
We should also set `Signature` field, right? as the first signature in sigs to be backwards compatible with previous charon versions.

Reply:
yeah, to ensure v0.18.0 is compatible with v0.17.0. Although, this can be removed when we release v0.19.0